lib/pharos/scripts/configure-firewalld.sh

```diff
@@ -7,11 +7,22 @@ if ! systemctl is-active --quiet firewalld; then
     systemctl start firewalld
 fi
 
-firewall-cmd --reload
+if ! firewall-cmd --info-service pharos-worker 2&>1 /dev/null ; then
```

Comment:
I don't get what this actually does. If the `pharos-worker` service does not exist in firewalld configs, then reload it? Later on there's the same condition used to actually load the `pharos-worker` service. :/

Reply:
We need to reload early because otherwise firewalld does not see those written config files and errors out.